app/javascript/home.js

```diff
@@ -0,0 +1,218 @@
+        'stylers': [
+          {
+            'visibility': 'simplified'
+          }
+        ]
+      },
+      {
+        'featureType': 'poi.business',
+        'elementType': 'labels.icon',
+        'stylers': [
+          {
+            'visibility': 'off'
+          }
+        ]
+      },
+      {
+        'featureType': 'road',
+        'stylers': [
+          {
+            'saturation': '-100'
+          }
+        ]
+      },
+      {
+        'featureType': 'road.arterial',
+        'stylers': [
+          {
+            'lightness': '30'
+          }
+        ]
+      },
+      {
+        'featureType': 'road.highway',
+        'stylers': [
+          {
+            'visibility': 'simplified'
+          }
+        ]
+      },
+      {
+        'featureType': 'road.local',
+        'stylers': [
+          {
+            'lightness': '40'
+          }
+        ]
+      },
+      {
+        'featureType': 'transit',
+        'stylers': [
+          {
+            'saturation': -100
+          }
+        ]
+      },
+      {
+        'featureType': 'water',
+        'elementType': 'geometry',
+        'stylers': [
+          {
+            'hue': '#ffff00'
+          },
+          {
+            'saturation': -97
+          },
+          {
+            'lightness': -25
+          }
+        ]
+      },
+      {
+        'featureType': 'water',
+        'elementType': 'labels',
+        'stylers': [
+          {
+            'saturation': -100
+          },
+          {
+            'lightness': -25
+          }
+        ]
+      }
+    ]
+  });
+
+  const infowindow = new google.maps.InfoWindow();
+
+  getJSON('./gardens', (data) => {
+    const gardens = [...Array(data.length)].map(_ => Array(4));
+    for (let i = 0; i < data.length; i++) {
+      gardens[i][0] = data[i].name;
+      gardens[i][1] = data[i].lat;
+      gardens[i][2] = data[i].long;
+      gardens[i][3] = `
+        <div class="card z-depth-0" style="width: 100%;">
+          <div class="card-image">
+            <img src="${data[i].image}">
+            <span class="card-title">
+              ${data[i].name}
+            </span>
+          </div>
+          <div class="card-content">
+            <p>${data[i].contact_name}</p>
+            <p> ${data[i].contact_number}</p>
+            <p>${data[i].address}</p>
+            <ul id=garden_id ${data[i].id} class="collection"></ul>
+          </div>
+        </div>`;
+
+      const marker = new google.maps.Marker({
+        position: new google.maps.LatLng(gardens[i][1], gardens[i][2]),
+        title: gardens[i][0],
+        map: map
+      });
+      google.maps.event.addListener(marker, 'click', (function(marker, i) {
+        return function() {
+          infowindow.setContent(gardens[i][3]);
+          infowindow.open(map, marker);
+          populate_produces();
+        };
+      })(marker, i));
+    }
+  }, (_) => {
+    alert('Something went wrong.');
+  });
+}
+
+export function populate_produces() {
```

Comment:
If we're keeping this function it would probably be smart to rename it so that its purpose is clearer. A change from produces to crops could help dramatically. A previous "populate" function includes the word table, maybe we could add the word list to make naming semi-consistent?